src/styles/main.scss

```diff
@@ -3,12 +3,28 @@
 
 #root {
     @import "containers/home";
+    background: $light-sea-green;
+}
+
+#outer-container {
```

Comment:
There is no reason for `#outer-container` over anything else. We just need to ensure we have the perspective property set on the parent of the menu and the page-wrapper. Feel free to refactor however you see fit @rebeccamadsen.